ihatemoney/forms.py

```diff
@@ -327,6 +327,7 @@ class BillForm(FlaskForm):
     what = StringField(_("What?"), validators=[DataRequired()])
     payer = SelectField(_("Who paid?"), validators=[DataRequired()], coerce=int)
     amount = CalculatorStringField(_("How much?"), validators=[DataRequired()])
+    is_reimbursement = BooleanField("Reimbursement", default=False)
```

Comment:
Add internationalisation call.
```suggestion
    is_reimbursement = BooleanField(_("Reimbursement"), default=False)
```